stdlib/distutils/dist.pyi

```diff
@@ -101,21 +103,21 @@ class Distribution:
     def print_commands(self) -> None: ...
     def get_command_list(self): ...
     def get_command_packages(self): ...
-    def get_command_class(self, command): ...
-    def reinitialize_command(self, command, reinit_subcommands: int = 0): ...
+    def get_command_class(self, command: str | Command) -> type[Command]: ...
```

Comment:
It doesn't look like this works with Command objects: https://github.com/python/cpython/blob/d025b1d2785ced8d436246aaf7627aa2c33493d4/Lib/distutils/dist.py#L807